build.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+# Use this with `npm run build` so that `./node_modules/.bin` are injected into
+# the PATH
+
+license='/*! algoliasearch 2.9.2 | © 2014, 2015 Algolia SAS | github.com/algolia/algoliasearch-client-js */'
```

Comment:
It doesn't seem there is an easy way to ask npm about the current package. Maybe I'm missing something though :/